tools/provision

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+BASEDIR=`dirname $0`/..
+
+if [ ! -d "$BASEDIR/.zulip-api-env" ]; then
+    virtualenv $BASEDIR/.zulip-api-env
+    echo "Virtualenv created."
+fi
+
+source $BASEDIR/.zulip-api-env/bin/activate
+
+if [ ! -f "$BASEDIR/.zulip-api-env/updated" -o $BASEDIR/requirements.txt -nt $BASEDIR/.zulip-api-env/updated ]; then
+    pip install -r $BASEDIR/requirements.txt
+    touch $BASEDIR/.zulip-api-env/updated
+    echo "Requirements installed."
+fi
```

Comment:
Let's diff with `requirements.txt` rather than using a timestamp; that's a bit safer if you're playing around with things.